tox.ini

```
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{37,38,39,310}-django{22,32}
+    py{37,38,39,310}-django{32}
```

```suggestion
    py{38}-django{32}
```
I'd limit this to the exact versions of Open Forms